packages/extension-browser/src/devtools/views/pages/config.tsx

```diff
@@ -89,6 +90,7 @@ const ConfigPage = ({ disabled, onStart }: Props) => {
             </Button>
             {' '}
             <PoweredBy className={styles.poweredBy} />
+            <Version />
```

Comment:
I think this may make more sense as a sub-components of `PoweredBy` so it shows everywhere we use that component.